common/config/rush/npm-shrinkwrap.json

```diff
@@ -2650,64 +2653,6 @@
       "resolved": "https://registry.npmjs.org/@types/z-schema/-/z-schema-3.16.31.tgz",
       "integrity": "sha1-LrHQCl5Ow/pYx2r94S4YK2bcXBw="
     },
-    "@uifabric/charting": {
```

Comment:
Something seems *really wrong* here... the @uifabric packages seemed to have disappeared when you ran this... not sure what happened??